uint8 ARM_TARGET=0
uint8 ARM_TRAJECTORY=1
uint8 type
ArmTarget armTarget
ArmTrajectory armTrajectory
GripperAction gripperAction